src/semidiscretization/semidiscretization_hyperbolic.jl

```diff
@@ -409,4 +409,20 @@ function rhs!(du_ode, u_ode, semi::SemidiscretizationHyperbolic, t)
 
     return nothing
 end
+
+function rhs!(du_ode, u_ode, u_global, semis,
```

Comment:
```suggestion
function rhs!(du_ode, u_ode, u_global, semis::SemidiscretizationCoupled,
```

Or at least add a comment that this targets `SemidiscretizationCoupled`?